jarbas/core/management/commands/companies.py

```diff
@@ -10,6 +10,10 @@
 
 class Command(LoadCommand):
     help = 'Load Serenata de Amor companies dataset into the database'
+    
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument('--batch-size', '-b', dest='batch_size', type=int, default=10000, help='Number of documents to be created at a time')
```

Comment:
Can we be a bit more friendly towards PEP8? We don't have to stop strictly at 79, but IMHO this line is too long…